experimental/dds/tree2/src/test/utils.ts

```diff
@@ -317,6 +317,15 @@ export class TestTreeProvider {
 		await summarizeNow(this.summarizer, "TestTreeProvider");
 	}
 
+	public async getSummaryInfo() {
```

Comment:
This method is exactly the same as the `summarize` method, but returns the summary info instead of void. This method is used to check if handles were used in the unit test added in this PR.

I would have just updated `summarize` to return the summary info, but I was not sure if there was a good reason behind having it return `void`.

Reply:
nah it's reasonable to make it return the summary for testing purposes, I'd just do that

Reply:
Removed `getSummaryInfo` and updated `summarize` to return summary info